scripts/proto-copy-cosmos-sdk.sh

```diff
@@ -8,12 +8,13 @@ find "$cosmos_protos" -maxdepth 1 -mindepth 1 \
   -exec sh -c 'file=$1; basename $1  | xargs -I % -n1 rm -rf "./third_party/proto/%"' _ {} \;
 cp -r "$cosmos_protos" ./third_party
 
-ibc_protos="$(go list -f '{{ .Dir }}' -m github.com/cosmos/ibc-go/v3)/proto"
+ibc_protos="$(go list -f '{{ .Dir }}' -m github.com/cosmos/ibc-go/v4)/proto"
 find "$ibc_protos" -maxdepth 1 -mindepth 1 \
   -exec sh -c 'file=$1; basename $1  | xargs -I % -n1 rm -rf "./third_party/proto/%"' _ {} \;
 cp -r "$ibc_protos" ./third_party
 
 chmod -R +w ./third_party/proto
 
-# Remove ICA protos
+# Remove ICA and fee IBC app protos
 rm -r ./third_party/proto/ibc/applications/interchain_accounts
+rm -r ./third_party/proto/ibc/applications/fee
```

Comment:
could u remind why remove ica and fee protos?

Reply:
Removed since we aren't using those apps, doesn't break anything if we kept it though